primitives/Cargo.toml

```diff
@@ -57,5 +57,4 @@ std = [
 ]
 runtime-benchmarks = [
 	"polkadot-parachain/runtime-benchmarks",
-	"frame-system/runtime-benchmarks",
```

Comment:
did you mean to remove this?

Reply:
I believe we had suspected that this could cause the benchmarks to fail...

Do you know if we need it?